### PR TITLE
feat: edit customer address

### PR DESCRIPTION
**Pull request by rob-mohc**

### CODE REVIEW

**Review by markbrownmohc — APPROVED** [review]

File: tags.go [hunk 1]
```diff
@@ -54,3 +54,14 @@ func GetTagValue(tags []string, key string, separator string) string {
 
 	return ""
 }
+
+// TagExists returns true if tag exists in tags
+func TagExists(tags Tags, tagValue string) bool {
```
Comment (ezelopes-mohc) [inline tags.go:59]: I would call this function `HasTag` instead 🙂

File: tags.go [hunk 2]
```diff
@@ -54,3 +54,14 @@ func GetTagValue(tags []string, key string, separator string) string {
...
+	for _, tag := range tags.Split() {
+		if strings.ToLower(tag) == strings.ToLower(tagValue) {
```
Comment (ezelopes-mohc) [inline tags.go:61]: I would create a new variable for `strings.ToLower(tagValue)` as we're running this function on a constant 🙂

File: customer-addresses.go [hunk 3]
```diff
@@ -6,6 +6,8 @@ type CustomerAddressRepository interface {
 	Create(id int64, address CustomerAddress) (CustomerAddress, error)
 	// Delete deletes the specified address for the specified customer
 	Delete(id int64, addressID int64) error
+	// Update updates an address for a specified customer
```
Comment (Ianmohc) [inline customer-addresses.go:9]:
```suggestion
	// Update updates the address for the specified customer
```

File: tags.go (hunk 1 above)
Comment (Ianmohc) [reply tags.go:59]: Can this not be a method also? You only need to pass the tagValue in then.

**Review by Ianmohc — CHANGES_REQUESTED** [review]

**Review by Ianmohc — APPROVED** [review]

**Pull request merged**